release_files/Dockerfile.redhat

```diff
@@ -38,43 +39,20 @@ LABEL "description"="OpenVINO(TM) Model Server is a solution for serving AI mode
 ARG INSTALL_RPMS_FROM_URL=
 ENV INSTALL_RPMS_FROM_URL=$INSTALL_RPMS_FROM_URL
 
-ARG INSTALL_DRIVER_VERSION="20.35.17767"
+ARG INSTALL_DRIVER_VERSION="21.49.21786"
```

Comment:
If we specify this argument the it should be used in this file. Currently it's defined but not used.

Reply:
Currently we use the ocl neo driver solution that installs newest available driver.
The solution with specifying direct driver version does not work.
It might be changed in future.